Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -421,6 +429,64 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         }
     }
 
+    private struct Imports: Decodable {
+        let imports: [String]
+    }
+
+    private func validateImports(
+        manifestPath: AbsolutePath,
+        toolsVersion: ToolsVersion,
+        callbackQueue: DispatchQueue,
+        completion: @escaping (Result<Bool, Error>) -> Void) {
```

Comment:
should this be `Result<Void, Error>`?

Reply:
Yah good point, was initially thinking about doing something with the bool, but ended up making the error state an actual thrown error to carry the list of violations.